pkg/version/release.go

```diff
@@ -6,7 +6,7 @@ package version
 var Version = "0.40.0"
 
 // PreReleaseID can be empty for releases, "rc.X" for release candidates and "dev" for snapshots
-var PreReleaseID = "dev"
+var PreReleaseID = "rc.0"
```

Comment:
we don't want this in main right?

Reply:
honest to god no idea 🤷‍♀️

Reply:
and some merge to main is failing below

Reply:
we definitely don't want this in main as we want to keep cutting dev releases by default